plugins/modules/gitlab_issue.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Ondrej Zvara (ozvara1@gmail.com)
+# Based on code:
+# Copyright (c) 2021, Lennert Mertens (lennert@nubera.be)
+# Copyright (c) 2021, Werner Dijkerman (ikben@werner-dijkerman.nl)
+# Copyright (c) 2015, Werner Dijkerman (ikben@werner-dijkerman.nl)
+# Copyright (c) 2019, Guillaume Martinez (lunik@tiwabbit.fr)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_issue
+short_description: Create, update, or delete GitLab issues
+version_added: '8.1.0'
+description:
+  - Creates an issue if it does not exist.
+  - When an issue does exist, it will be updated if the provided parameters are different.
+  - When an issue does exist and O(state=absent), the issue will be deleted.
+  - When multiple issues are detected, the task fails.
+  - Existing issues are matched based on O(title) and O(state_filter) filters.
+author:
+  - zvaraondrej (@zvaraondrej)
+requirements:
+  - Python >= 2.7
+  - python-gitlab >= 2.3.0
+extends_documentation_fragment:
+  - community.general.auth_basic
+  - community.general.gitlab
+  - community.general.attributes
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+
+options:
+  assignee_ids:
+    description:
+      - Comma separated list of assignees usernames omitting V(@) character.
+      - Set to empty string to unassign all assignees.
+    type: str
+  description:
+    description:
+      - A description of the issue.
+      - Gets overridden by a content of file specified at O(description_path), if found.
+    type: str
+  description_path:
+    description:
+      - A path of file containing issue's description.
+      - Accepts MarkDown formatted files.
+    type: path
+  issue_type:
+    description:
+      - Type of the issue.
+    default: issue
+    type: str
+    choices: ["issue", "incident", "test_case"]
+  labels:
+    description:
+      - Comma separated list of label names.
+    type: str
```

Comment:
Why not allow to pass a list in here?
```suggestion
    type: list
    elements: str
```